py3status/modules/prometheus.py

```diff
@@ -8,6 +8,8 @@
     format: Formatting of query result. Can refer to all labels from the query
         result. Query value placeholder __v.
         (default "{__v:.0f}")
+    format_down: Output if prometheus is not available
+        (default 'n/a')
```

Comment:
I don't use this module.

Does `format = "[{__v:.0f}]"` work ? It hides.
Does `format = "{__v:.0f}|\?show n/a"` work ? It shows `n/a`.  -- It's a replacement for `format_down`.

Reply:
`format = "{__v:.0f}|\?show n/a"` works! Thanks. Is this documented somewhere?
It only works in the module code, not from user config, right?

Reply:
This should work in user config too. Docs: https://py3status.readthedocs.io/en/latest/user-guide/configuration/#formatter

Reply:
If it doesn't work, then prometheus is not updated to py3status standards.

Reply:
The module has to be updated to use safe_format() for this to work it seems